### ISSUE TITLE
ReferenceError: fetch is not defined

### ISSUE THREAD

**Issue by Film-Worachai** [AidanWelch/google-translate-api#20]
```
/root/***/node_modules/google-translate-api-x/lib/defaults.cjs:9
        requestFunction: fetch,
                         ^

ReferenceError: fetch is not defined
    at Object.<anonymous> (/root/***/node_modules/google-translate-api-x/lib/defaults.cjs:9:19)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/root/***/node_modules/google-translate-api-x/lib/translate.cjs:2:29)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
```

**Comment by AidanWelch** [comment]
Addressed in d0c154333894405ffa9d09ba78910ea89f203514

If you are using an environment that doesn't have the Fetch api exposed you can pass any function that emulates the behavior of the Fetch api such as [node-fetch](https://www.npmjs.com/package/node-fetch) or even a simple wrapper around axios.  This is as simple as 

```js
import fetch from 'node-fetch';
import translate from 'google-translate-api-x';

const res = await translate('gato', {from: 'es', to: 'en', requestFunction: fetch});

console.log(res.text); // => 'cat'
```

**Comment by Film-Worachai** [comment]
Thank you.
